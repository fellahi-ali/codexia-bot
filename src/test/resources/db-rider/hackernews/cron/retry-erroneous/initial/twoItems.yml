hackernews_item:
    -
        external_id: 1
        type: ""
        by: author
        title: title
        url: url
        time: 2000-01-01 00:01:10
        deleted: false
    -
        external_id: 2
        type: ""
        by: author
        title: title
        url: url
        time: 2000-01-01 00:01:10
        deleted: false
