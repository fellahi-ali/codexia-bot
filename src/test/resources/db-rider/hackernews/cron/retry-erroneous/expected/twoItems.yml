hackernews_item:
    -
        external_id: 1
        type: story
        by: test1author111
        title: Title for test1_123
        url: https://example.com/test123
        time: 2009-01-15 06:12:49
        deleted: false
    -
        external_id: 2
        type: story
        by: test2author222
        title: Title for test2_222
        url: https://example.com/test456
        time: 2009-01-05 04:32:53
        deleted: false
